src/rules/checkLinesAlignment.js

```diff
@@ -0,0 +1,210 @@
+import {
+  set,
+} from 'lodash';
+import iterateJsdoc from '../iterateJsdoc';
+
+/**
+ * Aux method until we consider the dev envs support `String.prototype.matchAll` (Node 12+).
+ *
+ * @param {string}   string   String that will be checked.
+ * @param {RegExp}   regexp   Regular expression to run.
+ * @param {Function} callback Function to be called each iteration.
+ * @param {int}      limit    Limit of matches that we want to exec.
+ *
+ * TODO [engine:node@>=12]: Remove function and use `String.prototype.matchAll` instead.
+ */
+const matchAll = (string, regexp, callback, limit) => {
+  let result;
+  let index = 0;
+
+  while ((result = regexp.exec(string)) && index <= limit - 1) {
+    // eslint-disable-next-line promise/prefer-await-to-callbacks
+    callback(result, index++);
+  }
+};
+
+/**
+ * Get the full description from a line.
+ *
+ * @param {string} lineString The line string.
+ *
+ * @returns {string} The full description.
+ */
+const getFullDescription = (lineString) => {
+  return /(?:\S+\s+){4}(.*)/.exec(lineString)[1];
+};
+
+/**
+ * Get the expected positions for each part.
+ *
+ * @param {int[]} partsMaxLength Max length of each part.
+ * @param {int}   indentLevel    JSDoc indent level.
+ *
+ * @returns {int[]} Expected position for each part.
+ */
+const getExpectedPositions = (partsMaxLength, indentLevel) => {
+  // eslint-disable-next-line unicorn/no-reduce
+  return partsMaxLength.reduce(
+    (acc, cur, index) => {
+      return [...acc, cur + acc[index] + 1];
+    },
+    [indentLevel],
+  );
+};
+
+/**
+ * Check is not aligned.
+ *
+ * @param {int[]}   expectedPositions Expected position for each part.
+ * @param {Array[]} partsMatrix       Parts matrix.
+ *
+ * @returns {boolean}
+ */
+const isNotAligned = (expectedPositions, partsMatrix) => {
+  return partsMatrix.some((line) => {
+    return line.some(
+      ({position}, partIndex) => {
+        return position !== expectedPositions[partIndex];
+      },
+    );
+  });
+};
+
+/**
+ * Fix function creator for the report. It creates a function which fix
+ * the JSDoc with the correct alignment.
+ *
+ * @param {object}  comment           Comment node.
+ * @param {int[]}   expectedPositions Array with the expected positions.
+ * @param {Array[]} partsMatrix       Parts matrix.
+ * @param {RegExp}  lineRegExp        Line regular expression.
+ * @param {string}  tagIndentation    Tag indentation.
+ *
+ * @returns {Function} Function which fixes the JSDoc alignment.
+ */
+const createFixer = (comment, expectedPositions, partsMatrix, lineRegExp, tagIndentation) => {
+  return (fixer) => {
+    let lineIndex = 0;
+
+    // Replace every line with the correct spacings.
+    const fixed = comment.value.replace(lineRegExp, () => {
+      // eslint-disable-next-line unicorn/no-reduce
+      return partsMatrix[lineIndex++].reduce(
+        (acc, {string}, index) => {
+          const spacings = ''.padStart(expectedPositions[index] - acc.length, ' ');
```

Comment:
```suggestion
          const spacings = ' '.repeat(expectedPositions[index] - acc.length);
```

Whoops--sorry--I had forgotten about `repeat`. :-)

Reply:
Hum! Much better! \o/